kernel/uGraph.ml

```diff
@@ -39,7 +39,8 @@ type explanation =
   | Path of path_explanation
   | Other of Pp.t
 
-type univ_inconsistency = constraint_type * Sorts.t * Sorts.t * explanation option
+type univ_variable_printers = (Sorts.QVar.t -> Pp.t) * (Level.t -> Pp.t)
```

Comment:
We may want to use this type more often instead of passing 2 arguments but not in this PR